### PR TITLE
Login and Logout endpoints functional, front end /login page complete

### PR DESCRIPTION
**Pull request by kimberlym4488**
Please test :)

### CODE REVIEW

File: config/connection.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+require("dotenv").config();
 
 const sequelize = new Sequelize(
   process.env.DB_NAME,
   process.env.DB_USER,
   process.env.DB_PASSWORD,
   {
-    host: 'localhost',
-    dialect: 'mysql',
+    host: "localhost",
```
Comment (jmichaelbrown8) [inline config/connection.js:9]: I need to change this to "127.0.0.1" for it to run on my machine--does that still run for you on Windows?

File: public/js/login.js [hunk 2]
```diff
@@ -1,50 +1,49 @@
...
+  const username = document.querySelector("#username-signup").value.trim();
+  const password = document.querySelector("#password-signup").value.trim();
 
-  if (username && email && password) {
-    const response = await fetch('/api/users', {
-      method: 'POST',
-      body: JSON.stringify({ username, email, password }),
-      headers: { 'Content-Type': 'application/json' },
+  if (username && password) {
+    const response = await fetch("/api/users", {
```
Comment (jmichaelbrown8) [inline public/js/login.js:29]: The fetch endpoint here will will have to change to /api/user to match the route naming.

**Review by jmichaelbrown8 — COMMENTED** [review]
Great job! Just a couple of changes needed to get Mac development and sign up working. 

It would be cool to pass some of the validation details back to the user (like username must be between 8-15 characters, etc.) but not totally necessary.

File: config/connection.js (hunk 1 above)
Comment (jmichaelbrown8) [reply config/connection.js:9]: Nevermind, Zack just recommended we don't change this in code! Lo and I will just keep local changes to ourselves.

File: public/js/login.js (hunk 2 above)
Comment (lonewolfco) [reply public/js/login.js]: fixed!

**Review by lonewolfco — APPROVED** [review]

**Pull request merged**